projects/laji/src/app/+project-form/project-form.component.html

```diff
@@ -1,5 +1,9 @@
 <ng-container *ngIf="vm$ | async as vm; else loader">
-  <lu-sidebar *ngIf="!vm.form.options?.simple && !vm.form.options?.mobile; else simpleContent" [open]="(showNav$ | async)" [noPrint]="isPrintPage$ | async">
+  <lu-sidebar *ngIf="!vm.form.options?.simple && !vm.form.options?.mobile; else simpleContent"
+  [open]="(showNav$ | async)"
+  [noPrint]="isPrintPage$ | async"
```

Comment:
hmm can't pinpoint the exact spot but noPrint doesn't seem to [work here](https://176071265.dev.laji.fi/project/MHL.1/form/MHL.1/places/MNP.27299/print) but it's working in [beta](https://beta.laji.fi/project/MHL.1/form/MHL.1/places/MNP.27299/print). (Maybe merge dev branch here to make sure that the error is not caused changes in this branch)

Reply:
That doesn't seem to be working in `development` branch. The sidebar and the print button have `no-print` class, and emulating print mode in chrome hides those as it's supposed to. For some reason though, the sidebar & the print button are shown in the generated pdf... I have a vague memory that I've run into this issue before, but can't recall what the issue was.

Reply:
hmm it's working in dev, but this might be something to do with the build process in these task branches.